extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -670,7 +683,7 @@ export class AmpConsent extends AMP.BaseElement {
             // 1. Race condition on consent state change between schedule to
             //    display and display. Add one more check before display
             // 2. Should not schedule display with DISMISSED UNKNOWN state
-            this.scheduleDisplay_(instanceId);
+            return this.scheduleDisplay_(instanceId);
```

Comment:
Why we have the return here? I don't see the change to `this.scheduleDisplay_` return value.